modulo3/Clase21/Practica2-puntos3y4resueltos.sql

```diff
@@ -0,0 +1,9 @@
+
+/* punto 3 */
+
+SELECT * FROM northwind.products  where ProductID = 5 ORDER BY ProductName;
+SELECT * FROM northwind.products  where CategoryID = 4 ORDER BY ProductName;
+
```

Comment:
SELECT * FROM northwind.products  where SupplierID = 5 AND CategoryID = 4 ORDER BY ProductName;